README.md

```diff
@@ -215,13 +206,13 @@ Docker:
     cd example
     ```
 
-1. Create the `Dockerfile-x` file with `buildx` platform support:
+2. Create the `Dockerfile-x` file with `buildx` platform support:
```

Comment:
This change and the following change are not needed.  [Markdown ordered lists are valid](https://www.markdownguide.org/basic-syntax/#ordered-lists) if all items start with `1.`, which is our coding standard.
```suggestion
1. Create the `Dockerfile-x` file with `buildx` platform support:
```